lib/stylelintBaseConfig.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
I ended up not making any changes to the config files. The main change was updating the dependency versions to latest.